src/quo2/components/onboarding/small_option_card/view.cljs

```diff
@@ -35,35 +35,41 @@
      :subtitle subtitle}]])
 
 (defn- main-variant
-  [{:keys [title subtitle image]}]
+  [{:keys [title subtitle image max-height]}]
   [rn/view {:style style/main-variant}
    [rn/view {:style style/main-variant-text-container}
     [texts
      {:title    title
       :subtitle subtitle}]]
    [fast-image/fast-image
     {:accessibility-label :small-option-card-main-image
-     :style               {:flex 1}
+     :style               {:flex 1 :max-height max-height}
      :resize-mode         :contain
      :source              image}]])
 
 (defn small-option-card
   "Variants: `:main` or `:icon`"
-  [{:keys [variant title subtitle image on-press]
+  [{:keys [variant title subtitle image max-height on-press]
     :or   {variant :main}}]
   (let [main-variant?  (= variant :main)
-        card-component (if main-variant? main-variant icon-variant)]
+        card-component (if main-variant? main-variant icon-variant)
+        card-height    (if main-variant?
```

Comment:
I usually avoid nesting `if`s because they easily get hard to read. What do you think about using a `cond` instead?

Just as a suggestion:
```clojure
(cond
  (not main-variant?) style/icon-variant-height
  max-height          (min max-height style/main-variant-height)
  :else               style/main-variant-height)
```

Reply:
that does make sense! good suggestion, thanks